### ISSUE TITLE
Weird stuff getting dumped out

### ISSUE THREAD

**Issue by secrettriangle** [supercrabtree/k#38]
```
 19:05:41  dan.kaplun@N164MALT:~/asdf 
$ k
total 0
bar  foo
workon_cwd:2: scalar parameter WORKON_CWD created globally in function
workon_cwd:4: scalar parameter PROJECT_ROOT created globally in function
-rw-r--r-- 1 dan.kaplun staff 0 27 Apr   19:05   bar
bar  foo
workon_cwd:2: scalar parameter WORKON_CWD created globally in function
workon_cwd:4: scalar parameter PROJECT_ROOT created globally in function
-rw-r--r-- 1 dan.kaplun staff 0 27 Apr   19:05   foo
```

**Comment by supercrabtree** [comment]
Closed by ae05b486f2e30f9e9304b1bea743f66cfd80f329

**Comment by secrettriangle** [comment]
Confirmed, thanks!
